packages/sources/css/src/components/indicators/progressbar/src/index.css

```diff
@@ -41,6 +41,13 @@
   align-items: center;
 }
 
+/* Linear */
+.vtmn-progressbar_variant--linear .vtmn-progressbar_indicator {
+  --vtmn-progressbar_progress-transform: unset;
+
+  transform: var(--vtmn-progressbar_progress-transform);
```

Comment:
What do you think about used the second property from `var` to set a default value ? 
https://developer.mozilla.org/fr/docs/Web/CSS/var

Reply:
I think it's a good idea, I'll make the fix., thank you!
Just in case you didn't know, `var` already defaults to `unset` if the custom property doesn't exist. But in this specific case I think it is better to explicit this.